frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/homepage/MyTaskSubmissionsPortlet.java

```diff
@@ -60,8 +82,36 @@ public void doView(RenderRequest request, RenderResponse response) throws IOExce
             log.error("Could not fetch your moderations from backend", e);
         }
 
-        request.setAttribute(PortalConstants.MODERATION_REQUESTS, CommonUtils.nullToEmptyList(moderations));
+        JSONArray jsonModerations = getModerationData(moderations);
```

Comment:
Please have a look at the previous comment. Duplicated in the `MyTaskSubmissionsPortlet.java`.